ui/Screen/UserRegistration/PickHandleStep.svelte

```diff
@@ -3,7 +3,8 @@
   import validatejs from "validate.js";
 
   import * as notification from "../../src/notification.ts";
-  import * as user from "../../src/user.ts";
+  import * as id from "../../src/id.ts";
```

Comment:
Let's order imports alphanumerically.

Reply:
Is there a CI check that can run for these?